### DISCUSSION TITLE
Idea for a feature for adaptive refresh rate

### DISCUSSION THREAD

**Discussion by Medakekw** [tribalfs/GalaxyMaxHzPub#153] [Ideas]
Hey I got the app for my s21 fe to enable adaptive switching from 120 to 60hz, but I notice the color shift sometimes when the refresh rate changes.
Is this due to the brightness change due to refresh rate change on oleds or is it just that it goes from vivid to normal color(from display options).
If you could check this feature and potentially update it would be cool!

**Comment by tribalfs** [comment]
Hi! It's the first.  Pls check [FAQ#8](https://github.com/tribalfs/GalaxyMaxHzPub/wiki/Frequently-Asked-Questions-(FAQs)#8-why-sometimes-i-notice-my-color-shift-when-refresh-rate-switches-in-low-brightness-whats-the-workaround)
